### DISCUSSION TITLE
Sticky sidebar menu without QHeader

### DISCUSSION THREAD

**Discussion by LrxCyril** [elimintz/justpy#698] [Q&A] [answered]
Hi everybody,

I'm participating to develop a software using justpy and I manage the front of this one. 
Thanks a lot for this usefull framework.
I need to create a sidebar menu without header but the drawer don't stay fixe on the scroll when I don't use QHeader to define my sidebar.
In the case where the QHeader integrated, no problem, it's all right. The Drawer stay sticky.
Maybe isn't it possible to create a sidebar menu without QHeader component ?  

Thanks for your helping !

Case when sticky sidebar menu doesn't work
![sidebar_doesn-t_works](https://github.com/justpy-org/justpy/assets/16955374/1512e0f8-2e1f-46a6-b403-1a0e32b0b3da)
Case when sticky sidebar menu with Qheader works
![sidebar_with_header_works](https://github.com/justpy-org/justpy/assets/16955374/bd8384e0-d8f2-4ac5-8b97-deba64b34e00)

**Comment by WolfgangFahl** [comment, marked as answer]
You chances to get an answer might be higher at https://stackoverflow.com/questions/tagged/justpy also using the tag Quasar. Also you might wan to look how nicegui handles this - given that #685 is under discussion

**Reply by LrxCyril** [reply]
I will take a closer look at this solution and thanks for your fast answer.
